receiver/prometheusreceiver/internal/otlp_transaction.go

```diff
@@ -32,6 +32,10 @@ import (
 	"go.uber.org/zap"
 )
 
+const (
+	targetMetricName = "target_info"
```

Comment:
I may be misremembering, but I think this should be a metric named "target", of type info.

Reply:
So, we don't get type information passed down to the `Appender`. So we'll have to settle for comparing the metric name directly.

Reply:
The metric is still named "target" though, right?

Reply:
Umm, the metric name is `target_info`: https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#supporting-target-metadata-in-both-push-based-and-pull-based-systems

Should it be `target`?

Reply:
The MetricFamily name is `target`, but the MetricPoint name is `target_info`.  Do you know which applies here?  I had thought it was the MetricFamily name?

Reply:
It should be MetricPoint because we're dealing with `label.Labels` directly. We could also handle it inside `t.AddDataPoint` after we extra the metricFamily but it doesn't seem to offer any advantages imo.

Reply:
Thanks for explaining.